api/v1/task.proto

```diff
@@ -35,5 +35,5 @@ message Error {
 
     // Additional error information that the client code can use to handle
     // the error, such as retry delay or a help link.
-    repeated google.protobuf.Any details = 3;
+    repeated string details = 3;
```

Comment:
why this change? This should probably be best on its own commit.

Reply:
made it harder to test as a `google.protobuf.Any`

Reply:
Can you split this change out into its own commit. All the other changes in this commit make sense in that they are needed to do the testing, but this one just seems to be non-obvious. Splitting to its own commit you can add that info in.

Reply:
we talked and I said I was going to go back to `google.protobuf.Any details` but I've changed my mind. time constraints and ease (creating a pb.any type is a lot more involved than just a string slice) of testing I'm going to leave it as is.

Reply:
heh. then lets go back and do this one change in its own commit with this info

Reply:
updated